scripts/build/termux_create_pacman_subpackages.sh

```diff
@@ -121,15 +121,43 @@ termux_create_pacman_subpackages() {
 		# Allow packages to create arbitrary control files.
 		#termux_step_create_subpkg_debscripts
 
-		# Create the actual .deb file:
-		TERMUX_SUBPKG_PACMAN_FILE=$TERMUX_OUTPUT_DIR/${SUB_PKG_NAME}${DEBUG}-${TERMUX_PKG_FULLVERSION}-${SUB_PKG_ARCH}.pkg.tar.xz
+		# Configuring the selection of a copress for a batch.
+		case $PKG_COMPRESS in
```

Comment:
Please follow the general architecture of build-package.sh.

1. Rename PKG_COMPRESS variable to TERMUX_PACMAN_PACKAGE_COMPRESSION.
2. Where PKG_COMPRESS is first defined? I don't see any default defines but only a reference. https://github.com/termux/termux-packages/blob/master/scripts/build/termux_step_setup_variables.sh

Reply:
> Where PKG_COMPRESS is first defined?

It must be installed in the build.sh file.